src/Compilers/CSharp/Test/Symbol/Symbols/StaticAbstractMembersInInterfacesTests.cs

```diff
@@ -33330,5 +33330,36 @@ public class C5 : I1<C5>
                     );
             }
         }
+
+        [Fact, WorkItem(65685, "https://github.com/dotnet/roslyn/issues/65685")]
+        public void NoDuplicateConversionForImplicitAndExplicit()
+        {
+            CreateCompilation("""
+                interface ISam1<T> where T : ISam1<T>
+                {
+                    static abstract T operator +(T a, T b);
+                    static abstract implicit operator int(T t);
+                }
+
+                interface ISam2<T> where T : ISam2<T>
+                {
+                    static abstract T operator +(T a, T b);
+                    static abstract implicit operator int(T t);
+                }
+
+                class Foo : ISam1<Foo>, ISam2<Foo>
```

Comment:
>class Foo [](http://example.com/codeflow?start=16&length=9)

To make sure behavior is not order dependent, let's add another class with methods of interest declared in different order.